dbtmetabase/metabase.py

```diff
@@ -139,7 +164,13 @@ def models_compatible(self, database_id: str, models: Sequence) -> bool:
 
         return are_models_compatible
 
-    def export_models(self, database: str, models: Sequence[MetabaseModel], aliases):
+    def export_models(
+        self,
+        database: str,
+        models: Sequence[MetabaseModel],
+        aliases,
+        **kwargs,
```

Comment:
This is why I want to move to a `Config` object, kwargs everywhere will be a pain to debug and click through in people's IDEs when they're figuring out how it works.

Reply:
Agreed with the config obj, will be a much cleaner method